src/builtins/builtins-wasm-gen.cc

```diff
@@ -110,6 +111,46 @@ TF_BUILTIN(WasmGrowMemory, WasmBuiltinsAssembler) {
   ReturnRaw(Int32Constant(-1));
 }
 
+TF_BUILTIN(WasmToBigIntJavaScript_WasmRuntimeTrampoline,
+           WasmBuiltinsAssembler) {
```

Comment:
Style nit: It looks like src/builtins/builtins-wasm-gen.cc has lots of these trampolines, but never uses "trampoline" in the name. Would it make sense to copy the style of the surrounding code, and avoid calling it "trampoline" here, and keeping the related built-in in a separate file alongside the others?

Reply:
I agree, I did this to avoid the collision with the builin under this one called `WasmToBigIntJavaScript`. I think that something is wrong here, i'll work on it.